apps/scan/backend/src/scanners/custom/state_machine.ts

```diff
@@ -312,13 +312,14 @@ async function reset({ client }: Context): Promise<void> {
 async function scan({ client, workspace }: Context): Promise<SheetOf<string>> {
   assert(client);
   debug('Scanning');
-  const isUltrasonicDisabled = workspace.store.getIsUltrasonicDisabled();
+  const isMultiSheetDetectionDisabled =
+    workspace.store.getIsMultiSheetDetectionDisabled();
   const scanResult = await client.scan({
     wantedScanSide: ScanSide.A_AND_B,
     resolution: ImageResolution.RESOLUTION_200_DPI,
     imageColorDepth: ImageColorDepthType.Grey8bpp,
     formStandingAfterScan: FormStanding.HOLD_TICKET,
-    doubleSheetDetection: isUltrasonicDisabled
+    doubleSheetDetection: isMultiSheetDetectionDisabled
```

Comment:
While we're unifying naming, what do you think of "double sheet" vs. "multi sheet"?

Reply:
I think double sheet is a little clearer, but chose multi sheet because I've been seeing the MSD acronym used a lot in discussion, which made me think that was the term of art. But I don't feel strongly and would be down to standardize on either. What do you think?

Reply:
I think MSD is fine. The question here was really just "do you think we should also change double to multi?"

Reply:
I almost changed it, but decided since it was full encapsulated within the custom scanner code, it wasn't a high value change moving forward